### PR TITLE
Don't fetch system groups etc, because they don't exist in the schema

### PR DESCRIPTION
**Pull request by thomax**
…and thus will cause the Studio to crash

### CODE REVIEW

File: template/MyTool.js [hunk 1]
```diff
@@ -12,6 +12,13 @@ import schema from 'part:@sanity/base/schema'
 // See https://github.com/css-modules/css-modules for more info.
 import styles from './MyTool.css'
 
+function getLocalSchemaTypeNames() {
```
Comment (bjoerge) [inline template/MyTool.js]: Maybe rename to `getDocumentTypeNames`?

**Review by bjoerge — APPROVED** [review]
Just a minor suggestion, otherwise lgtm

**Review by rexxars — APPROVED** [review]
LGTM

File: template/MyTool.js (hunk 1 above)
Comment (thomax) [reply template/MyTool.js]: Sure 👍

**Pull request merged**